src/ControlTasks/time_manager.py

```diff
@@ -0,0 +1,40 @@
+from .control_task_base import ControlTaskBase
+from ..sfr import StateFieldRegistry
+
+import time
+# control tasks init and sfr init
+
+
+class TimeManager(ControlTaskBase):
+
+    def setup(self):
+        pass
+
+    def default(self):
+        self.sfr.set("start_time", time.time())
+        self.sfr.set("target_control_cycle_duration", 1)
+
+    def execute(self):
+
+        # add start time into sfr
+        # get start time after all control task calls
+        current_time = time.time()
+        start_time = self.sfr.get("start_time")
+        runtime = self.sfr.get("target_control_cycle_duration")
```

Comment:
let's change the runtime variable to `target_control_cycle_duration` or something more descriptive

Further this is a variable that we don't expect to change during the duration of a given run, let's pull this from the config file instead.